phys/module_diagnostics_driver.F

```diff
@@ -305,6 +313,13 @@ SUBROUTINE diagnostics_driver ( grid, config_flags,                  &
                ,vmet=grid%vmet                                      &
                ,speed=grid%speed                                    &
                ,dir=grid%dir                                        &
+               ,psfc=grid%psfc                                      &
+               ,rain=grid%rain                                      &
+               ,liqrain=grid%liqrain                                &
+               ,tpw=grid%tpw                                        &
+               ,psfc2d=grid%psfc2d                                        &
```

Comment:
@smileMchen 
Ming,
Line up the continuations

Reply:
@davegill 
Dave,
As suggested, tt has been lined up.